SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1001,6 +1001,7 @@ void USpatialReceiver::OnCommandRequest(Worker_CommandRequestOp& Op)
 	else if (Op.request.component_id == SpatialConstants::RPCS_ON_ENTITY_CREATION_ID && CommandIndex == SpatialConstants::CLEAR_RPCS_ON_ENTITY_CREATION)
 	{
 		Sender->ClearRPCsOnEntityCreation(Op.entity_id);
+		Sender->SendEmptyCommandResponse(SpatialConstants::RPCS_ON_ENTITY_CREATION_ID, SpatialConstants::CLEAR_RPCS_ON_ENTITY_CREATION, Op.request_id);
```

Comment:
Could we call it with Sender->SendEmptyCommandResponse(Op.request.component_id, CommandIndex, Op.request_id)? That way it will be same as in other places below. Or you prefer to keep this particular case explicit?

Reply:
Ah for some reason I thought CommandIndex didn't exist. Can fix.